nix/lib/toolchains.nix

```diff
@@ -1,17 +1,21 @@
-{
```

Comment:
Formatting changes here is likely due to using a different formatter from the one `treefmt` is using.

Reply:
I think this is using the one rnix-lsp provides, I'll run treefmt 👍